src/commands/test/examples/broken/test_basic_broken.cairo

```diff
@@ -7,6 +7,6 @@ func test_broken_contract{syscall_ptr : felt*, range_check_ptr}():
     alloc_locals
 
     local contract_address
-    %{ id.contract_address = deploy_contract("src/commands/test/examples/broken/basic_broken.cairo") %}
+    %{ id.contract_address = deploy_contract("src/commands/test/examples/broken/basic_broken.cairo").contract_address %}
```

Comment:
I'd change `contract_address` to `address`. It's hard to confuse it with a different type of `address`, it's easier to type, and this property will often be used.

Reply:
It is cairo internal object, I don't like the idea of mapping it because of slightly simpler name